Jenkinsfile1

```diff
@@ -0,0 +1,23 @@
+pipeline {
+    agent any
+    tools{
+    maven 'mv'
+    }
+    stages {
+        stage("git") {
+            steps {
```

Comment:
delete stage git

Reply:
Do we use anywhere UserDao method getRandom?